internal/v2/remote_signer/remote_signer.go

```diff
@@ -0,0 +1,12 @@
+// Remote signer library that offloads private key operations to S2Av2.
+package remotesigner
+
+import (
+	"crypto"
+	"crypto/tls"
+)
+
+// NewRemoteSigner returns an instance of RemoteSigner, an implemenation of the crypto.Signer interface.
```

Comment:
nit: typo on spelling of "implementation" and please add a line break if the comment line exceeds 80 characters.